pages/[chain]/collection/[contract].tsx

```diff
@@ -496,9 +497,7 @@ const CollectionPage: NextPage<Props> = ({ id, ssr }) => {
                             as="h6"
                             css={{ color: '$bg', fontWeight: 900 }}
                           >
-                            {`${nativePrice?.toFixed(2)} ${
-                              chainCurrency.symbol
-                            }`}
+                            {formatBN(nativePrice, 4)} {chainCurrency.symbol}
```

Comment:
I think we should use the FormatCrypto component

Reply:
Resolved.